src/recipe.rs

```diff
@@ -150,7 +154,10 @@ impl<'src, D> Recipe<'src, D> {
       }
       let mut evaluated = String::new();
       let mut continued = false;
-      let quiet_command = lines.peek().map_or(false, |line| line.is_quiet());
+
+      let quiet_command = self.no_quiet() && context.settings.quiet
```

Comment:
This is used to determine if we should strip a leading `@` from the line, so we should only do so if it starts with an `@`:

```suggestion
      let quiet_line = lines.peek().map_or(false, |line| line.is_quiet());
```

Reply:
done